src/platform/forms/address/helpers.js

```diff
@@ -53,7 +53,10 @@ export function isEmptyAddress(address) {
  * @param {string} abbreviation
  * @returns {string}
  */
-export function getStateName(abbreviation = '') {
+export function getStateName(abbreviation) {
+  if (!abbreviation) {
+    return abbreviation;
```

Comment:
I think this changes the return value for an invalid `STATE_NAME` from a falsey `undefined` to a truthy string.

Reply:
Hmmm, just just adds proper handling for passing in `null` for `abbreviation`. Previously this function was getting called with `null` and it would blow up when it tried to call `toUpperCase()`. Now when you have a state code that's `null` the state name returned from this function will also be `null`.

Reply:
But we also changed the code so this should no longer even get called with a `null` value, but I wanted a belt and suspenders approach to this fix